components/prize_pool/wikis/fifa/prize_pool_custom.lua

```diff
@@ -0,0 +1,64 @@
+---
+-- @Liquipedia
+-- wiki=fifa
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+
+local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector', {requireDevIfEnabled = true})
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+local TIER_VALUE = {8, 4, 2}
+
+-- Template entry point
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+	args.opponentLibrary = 'Opponent/Custom'
```

Comment:
```suggestion
```

Reply:
Does this mean that the Opponent/Custom calls inside the **Module:info** is to be kept (and because of that, this line no longer needs on prize pool if I understand that correctly?)

Reply:
`args.opponentLibrary = 'Opponent/Custom'` was deprecated some weeks ago
it does nothing as we load the correct opponent module via the one set in module info